### PR TITLE
Add PackageManagerTabs to `guides/deploy/vercel.mdx`

### PR DESCRIPTION
**Pull request by trueberryless**
#### Description (required)

Add pnpm and yarn commands via PackageManagerTabs component.

!!! IMPORTANT !!!

I'm working from mobile, so commands are not tested, not sure if pnpm and yarn `add` is correct. Pls review!

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/en/guides/deploy/vercel.mdx [hunk 1]
```diff
@@ -78,10 +93,26 @@ After your project has been imported and deployed, all subsequent pushes to bran
...
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm add -g vercel
+      vercel
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn add -g vercel
```
Comment (yanthomasdev) [inline src/content/docs/en/guides/deploy/vercel.mdx:111]: There's no `-g` flag in yarn, instead its prefixed by global
```suggestion
yarn global add vercel
```

File: src/content/docs/en/guides/deploy/vercel.mdx
[the remark this answers is not in the document]
Comment (trueberryless) [reply src/content/docs/en/guides/deploy/vercel.mdx:1]: Sure, I haven't seen this one thanks!

**Comment by trueberryless** [comment]
> Just a heads up that in future, if this isn't a super urgent fix, then it would be more helpful if you waited to PR until you knew what you were adding were correct! 😅

Yeah, I'm really sorry about. I'm just too enthusiastic about Astro, so I work from mobile... Gonna stop such things in the future!

> Have you had a chance to verify the content?

I guess I could have asked Mr Google what the pnpm and yarn equivalents are...

File: src/content/docs/en/guides/deploy/vercel.mdx [hunk 2]
```diff
@@ -27,18 +28,46 @@ To enable SSR in your Astro project and deploy on Vercel:
...
+  ```shell
+  pnpm dlx astro add vercel
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn dlx astro add vercel
+  ```
+  </Fragment>
+</PackageManagerTabs>
```
Comment (TheOtterlord) [inline src/content/docs/en/guides/deploy/vercel.mdx:47]: `dlx` isn't required for pnpm/yarn. removing it to sync the syntax with our vercel integration guide.
``````suggestion
<PackageManagerTabs>
  <Fragment slot="npm">
  ```shell
  npx astro add vercel
  ```
  </Fragment>
  <Fragment slot="pnpm">
  ```shell
  pnpm astro add vercel
  ```
  </Fragment>
  <Fragment slot="yarn">
  ```shell
  yarn astro add vercel
  ```
  </Fragment>
</PackageManagerTabs>
``````

**Review by TheOtterlord — APPROVED** [review]
lgtm. one small change to sync the commands with our vercel integration guide.

**Pull request merged**